src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+using System;
+using System.Resources;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Azure .NET SDK")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+[assembly: AssemblyTitle("Microsoft Azure SQL Server Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SQL.")]
+[assembly: AssemblyVersion("0.10.0.0")]
+[assembly: AssemblyFileVersion("0.51.0.0")]
```

Comment:
We can still mark the NuGet package as "-prerelease" until all the swagger conversions are done.

The main motivation behind this is to make a distinction between hyak generated and swagger generated nuget packages.